code/extensions/CwpSiteSummaryExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Extends the site summary report to list the appropriate versions in the report header
+ */
+class CwpSiteSummaryExtension extends Extension
+{
+
+    /**
+     * Updates the modules used for the version label by:
+     *  - Removing SS Framework
+     *  - Adding CWP
+     *  - Relabelling SS CMS
+     *
+     * @param $modules
+     */
+    public function updateVersionModules(&$modules)
+    {
+        unset($modules['silverstripe/framework']);
+        $modules = ['cwp/cwp' => 'CWP'] + $modules;
```

Comment:
Oh one more - cwp/cwp is mandatory in CWP 1.x but it's not in CWP 2.x - can you please use cwp/cwp-core instead? They'll share the same version numbers so it should be a drop in replacement